Bio/SeqIO/UniprotIO.py

```diff
@@ -355,14 +355,23 @@ def _parse_dbReference(element):
             self.ParsedSeqRecord.dbxrefs.append(
                 element.attrib["type"] + ":" + element.attrib["id"]
             )
-            # e.g.
-            # <dbReference type="PDB" key="11" id="2GEZ">
-            #   <property value="X-ray" type="method"/>
-            #   <property value="2.60 A" type="resolution"/>
-            #   <property value="A/C/E/G=1-192, B/D/F/H=193-325" type="chains"/>
-            # </dbReference>
             if "type" in element.attrib:
-                if element.attrib["type"] == "PDB":
+                # <dbReference type="EMBL" id="U96180">
+                #    <property type="protein sequence ID" value="AAB66902.1"/>
+                #    <property type="molecule type" value="mRNA"/>
+                # </dbReference>
+                if element.attrib['type'] == 'EMBL':
```

Comment:
We've started enforcing double quotes as the preferred style in Biopython, thus:

```
$ flake8
./Tests/test_Uniprot.py:423:1: D400 First line should end with a period
./Bio/SeqIO/UniprotIO.py:363:35: Q000 Remove bad quotes
./Bio/SeqIO/UniprotIO.py:363:46: Q000 Remove bad quotes
./Bio/SeqIO/UniprotIO.py:365:28: Q000 Remove bad quotes
./Bio/SeqIO/UniprotIO.py:365:61: Q000 Remove bad quotes
./Bio/SeqIO/UniprotIO.py:366:51: Q000 Remove bad quotes
./Bio/SeqIO/UniprotIO.py:366:62: Q000 Remove bad quotes
./Bio/SeqIO/UniprotIO.py:367:69: Q000 Remove bad quotes
./Bio/SeqIO/UniprotIO.py:367:102: Q000 Remove bad quotes
```